youtube_dl/extractor/platzi.py

```diff
@@ -186,10 +176,12 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, course_name)
 
+        initialData = self._search_regex(
+            (r'window.initialData\s*=\s*({.+?})\s*;\s*\n', r'window.initialData\s*=\s*({.+?})\s*;'),
+            webpage, 'window.initialData')
         props = self._parse_json(
```

Comment:
This would fail more gracefully:
```
        props = self._parse_json(
            self._search_regex(r'data\s*=\s*({.+?})\s*;', webpage, 'data'),
            course_name)
        props = try_get(props, lambda x: x['initialProps'], dict) or {}
```